### PR TITLE
Revert "18292 - switch to use quay copies of... (#702)"

### PR DESCRIPTION
**Pull request by benoitf**
### What does this PR do?
This is a partial revert for commit 2ce2168579106f8a63d55077a22054321d5f9845.

### What issues does this PR fix or reference?
Using docker login should allow to make releases
We can ask upstream projects to move their images on other docker registries in the meantime

### How to test this PR?
All plug-ins work as before

### PR Checklist

[As the author of this Pull Request I made sure that:](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#pull-request-template-and-its-checklist)

- [x] [The Eclipse Contributor Agreement is valid](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#the-eclipse-contributor-agreement-is-valid)
- [x] [Code produced is complete](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-produced-is-complete)
- [x] [Code builds without errors](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-builds-without-errors)
- [x] [Tests are covering the bugfix](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#tests-are-covering-the-bugfix)
- [x] [The repository devfile is up to date and works](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#the-repository-devfile-is-up-to-date-and-works)
- [x] [Sections `What issues does this PR fix or reference` and `How to test this PR` completed](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#sections-what-issues-does-this-pr-fix-or-reference-and-how-to-test-this-pr-completed)
- [x] [Relevant user documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [x] [Relevant contributing documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [x] [CI/CD changes implemented, documented and communicated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#cicd-changes-implemented-documented-and-communicated)

### Reviewers

Reviewers, please comment how you tested the PR when approving it.

@nickboldt needs to be backported to 7.22.x branch and included in 7.22.0 tag

### CODE REVIEW

File: v3/plugins/cdr/code-server/3.6.2/meta.yaml [hunk 1]
```diff
@@ -5,11 +5,11 @@ version: 3.6.2
 type: Che Editor
 displayName: code-server
 title: Visual Studio Code - Web (code-server)
-description: An open source distribution of Visual Studio Code as a cloud IDE
+description: A open source distribution of Visual Studio Code as a cloud IDE
```
Comment (nickboldt) [inline v3/plugins/cdr/code-server/3.6.2/meta.yaml:8]: you undid a typo correction?

File: v3/plugins/cdr/code-server/2.1650-vsc1.39.2/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,31 @@
+apiVersion: v2
```
Comment (nickboldt) [inline v3/plugins/cdr/code-server/2.1650-vsc1.39.2/meta.yaml:1]: the 2.x versions of cdr/code-server are obsoleted by version 3.6.2. Why do we want them back?

File: v3/plugins/redhat/mta/0.0.46/meta.yaml [hunk 3]
```diff
@@ -23,7 +23,7 @@ spec:
     attributes:
       protocol: http
   containers:
-  - image: "quay.io/windupeng/mta-vscode-extension:latest"
+  - image: "docker.io/windup3/rhamt-vscode-extension:java8"
```
Comment (nickboldt) [inline v3/plugins/redhat/mta/0.0.46/meta.yaml:26]: why undo this change? @johnsteele specifically told me to use this replacement image since HE put it there. Upstream for the win, right?

**Review by nickboldt — CHANGES_REQUESTED** [review]
at least 2 things are amiss with this revert PR. See inline.

File: v3/plugins/cdr/code-server/2.1650-vsc1.39.2/meta.yaml (hunk 2 above)
Comment (benoitf) [reply v3/plugins/cdr/code-server/2.1650-vsc1.39.2/meta.yaml:1]: because you did a PR named "switch to quay.io" and I revert this one.
if you wanted to cleanup, then it should have been a separate commit or a separate PR or never been approved/merged

File: v3/plugins/cdr/code-server/next/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+  endpoints:
+   -  name: "code-server"
+      public: true
+      targetPort: 9000
+      attributes:
+        protocol: http
+        type: ide
+  containers:
+   - name: code-server
+     image: "docker.io/chinodesuuu/coder:next"
```
Comment (nickboldt) [inline v3/plugins/cdr/code-server/next/meta.yaml:23]: latest images are now on docker.io/codercom/code-server, which is why @ericwill suggested we could just remove this entirely and keep a single versioned version (3.6.2) until we move to the latest 3.7 one.

File: v3/plugins/cdr/code-server/3.6.2/meta.yaml (hunk 1 above)
Comment (benoitf) [reply v3/plugins/cdr/code-server/3.6.2/meta.yaml]: so please next time, do separate commits when you do something not related to the title of the PR
because here it's highlighting that you did changes in a commit that was not related to update to quay.io

File: v3/plugins/cdr/code-server/2.1650-vsc1.39.2/meta.yaml (hunk 2 above)
Comment (nickboldt) [reply v3/plugins/cdr/code-server/2.1650-vsc1.39.2/meta.yaml]: yeah I guess the conversation between Eric and I last night before I submitted the PR based on his recommendations wasn't captured in the PR itself.

File: v3/plugins/cdr/code-server/next/meta.yaml (hunk 4 above)
Comment (benoitf) [reply v3/plugins/cdr/code-server/next/meta.yaml:23]: he never suggested that in the PR, the PR doesn't mention that as well
Not written : never happened !

**Review by ericwill — APPROVED** [review]

**Review by nickboldt — APPROVED** [review]
